katalog/nginx/bases/controller/DaemonSet-ingress-nginx-controller.yml

```diff
@@ -37,27 +55,26 @@ spec:
                   - /wait-shutdown
           args:
             - /nginx-ingress-controller
-            - --configmap=$(POD_NAMESPACE)/nginx-configuration
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx
-            - --annotations-prefix=nginx.ingress.kubernetes.io
-            - --http-port=8080
-            - --https-port=8443
```

Comment:
why did we delete these 2 args? without these args NGINX will listed in ports 80 and 443 instead of 8080 and 8443, breaking the service. Did you forget to adjust the ports below?

Reply:
My bad, I thought the values I set would put the port in args also, let me fix that!